front/app/containers/LandingPage/index.tsx

```diff
@@ -60,256 +35,39 @@ const Container = styled.main`
-
-                <FeatureFlag name="initiatives">
-                  {postingProposalsEnabled && <StyledInitiativesCTABox />}
-                </FeatureFlag>
-              </StyledContentContainer>
-
-              {showCustomSection && (
-                <CustomSectionContentContainer>
-                  <StyledQuillEditedContent>
-                    <Fragment
-                      name={
-                        !isNilOrError(homepageInfoPage)
-                          ? `pages/${
-                              homepageInfoPage && homepageInfoPage.id
-                            }/content`
-                          : ''
-                      }
-                    >
-                      <T value={customSectionBodyMultiloc} supportHtml={true} />
-                    </Fragment>
-                  </StyledQuillEditedContent>
-                </CustomSectionContentContainer>
-              )}
-
-              {!authUser && (
-                <FooterBanner>
-                  {headerSloganMultiLoc ? (
-                    <T value={headerSloganMultiLoc}>
-                      {(translatedSlogan) =>
-                        translatedSlogan ? (
-                          <h2>{translatedSlogan}</h2>
-                        ) : (
-                          genericSlogan
-                        )
-                      }
-                    </T>
-                  ) : (
-                    genericSlogan
-                  )}
-                  {displayHeaderAvatars && <StyledAvatarBubbles />}
-                  <Button
-                    fontWeight="500"
-                    padding="13px 22px"
-                    buttonStyle="primary-inverse"
-                    onClick={this.signUpIn}
-                    text={<FormattedMessage {...messages.createAccount} />}
-                  />
-                </FooterBanner>
-              )}
-              <CityLogoSection />
-            </Content>
-          </Container>
-        </>
-      );
-    }
-
-    return null;
-  }
-}
-
-const Data = adopt<DataProps, InputProps>({
-  locale: <GetLocale />,
-  tenant: <GetAppConfiguration />,
-  authUser: <GetAuthUser />,
-  homepageInfoPage: <GetPage slug="homepage-info" />,
-  postingPermission: <GetInitiativesPermissions action="posting_initiative" />,
-});
-
-const LandingPageWithHoC = withTheme(LandingPage);
-
-// TODO: add spinner fallback for lazy-loaded cards?
+const LandingPage = ({}: Props) => {
+  const authUser = useAuthUser();
+
+  return (
+    <>
+      <Container id="e2e-landing-page">
+        {!isNilOrError(authUser) ? (
+          <SignedInHeader />
+        ) : (
+          <Fragment name="signed-out-header">
+            <SignedOutHeader />
+          </Fragment>
+        )}
+
+        <Content>
+          <Suspense fallback={<LoadingBox />}>
+            <MainContent />
+            <HomepageInfoSection />
+            <Footer />
+          </Suspense>
+        </Content>
```

Comment:
Performance improvement we can make on many pages.

Did this initially to compensate for the increased load above the fold with the 3 layouts that will load in the SignedOutHeader component if the module is turned on.